manager/src/test/resources/cloud_providers/cloud_providers.json

```diff
@@ -0,0 +1,16 @@
+[
+  {
+    "id": "aws",
+    "name": "aws",
+    "display_name": "Amazon Web Services",
+    "enabled" : "true",
+    "regions": [
+      {
+        "id": "us-east-1",
+        "name": "us-east-1",
+        "display_name": "US East, N. Virginia",
+        "enabled": "true"
+      }
+    ]
+  }
+]
```

Comment:
why do you need the same `cloud_providers.json` in the test resources?

Reply:
Left-over from testing. Removed.